cmd/snapd/main.go

```diff
@@ -31,14 +31,14 @@ import (
 	"github.com/snapcore/snapd/logger"
 	"github.com/snapcore/snapd/osutil"
 	"github.com/snapcore/snapd/sandbox"
-	"github.com/snapcore/snapd/sanity"
 	"github.com/snapcore/snapd/snapdenv"
 	"github.com/snapcore/snapd/snapdtool"
 	"github.com/snapcore/snapd/systemd"
+	"github.com/snapcore/snapd/validity"
 )
 
 var (
-	sanityCheck = sanity.Check
+	validityCheck = validity.Check
```

Comment:
Looking into the `sanity` package, I think "precondition" might be more appropriate since the checks asserts something about the system before snapd can run normal operations